network-proto/node.proto

```diff
@@ -100,6 +100,9 @@ message GossipMessage {
     bytes content = 2;
 }
 
+message ConfirmAnnounceBlock {}
+message ConfirmAnnounceTx {}
```

Comment:
I'd just keep to the pattern elsewhere (picked up from gRPC guides) and name them
`AnnounceBlockResponse` and `AnnounceTransactionResponse`.